app/controllers/users_controller.rb

```diff
@@ -19,4 +21,29 @@ def other_instances
     tenant = current_tenant
     ActsAsTenant.without_tenant { Instance.containing_user(@user) - [tenant] }
   end
+
+  def destroy
+    if @course_user.student?
+      @course_user.destroy
+      respond_to do |format|
+        format.html { redirect_to course_users_path(@course), notice: 'Student was successfully removed from the course.' }
+        format.json { head :no_content }
+      end
+    else
+      respond_to do |format|
+        format.html { redirect_to course_users_path(@course), alert: 'Only students can be removed.' }
+        format.json { render json: { error: 'Only students can be removed' }, status: :unprocessable_entity }
+      end
+    end
+  end
```

Comment:
Deletion should apply for _any_ course_user, not just student.  
There is no business logic which says only students can be removed.